packages/api/src/hooks/useAccesiblePlatforms.ts

```diff
@@ -0,0 +1,32 @@
+import { useMemo } from 'react';
+import useLandingCompany from './useLandingCompany';
+
+/**
+ * A custom hook that provides flags to determine the accessibility status of cTrader and Dxtrade based on the current country of residence.
+ */
+const useAccesiblePlatforms = () => {
+    const { data: landing_company, ...rest } = useLandingCompany();
+
+    const modified_accesible_platform = useMemo(() => {
+        if (!landing_company) return;
+
+        /** check if ctrader jurisdiction is offered in the landing_company response  */
+        const is_ctrader_available = landing_company?.ctrader?.all?.standard === 'svg';
+        /** check if dxtrade is in the landing_company response */
+        const is_dxtrade_available = landing_company?.dxtrade_all_company;
+
+        return {
+            /** is ctrader accessible for this country of residence */
+            isCtraderAvailable: !!is_ctrader_available,
```

Comment:
if it is boolean value or const it should follow camelCase same as other files
```suggestion
            !!is_ctrader_available,
```